src/applications/ezr/tests/unit/utils/helpers/file-attachments.unit.spec.js

```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai';
+import {
+  createPayload,
+  parseResponse,
+} from '../../../../utils/helpers/file-attachments';
+
+describe('ezr `createPayload` method', () => {
+  let mockFile;
+
+  beforeEach(() => {
```

Comment:
Is there a specific reason to create this `mockFile` in a before block? Seems like you could just create a const in the it block?

Reply:
I'm not sure. I pulled this code from the EZ test suite and just plugged it into the EZR. I'll take a look. Thanks!